lib/aws/rails/railtie.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Aws
+  # Use the Rails namespace.
+  module Rails
+    # @api private
+    class Railtie < ::Rails::Railtie
+      initializer 'aws-sdk-rails.initialize',
```

Comment:
Not related to this change, but agree we should move each to a seperate initializer.

Reply:
Agreed, but I need to know which initializer should go when.